pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -490,6 +491,28 @@ monitorLoop:
 	}
 }
 
+func (l *LibvirtDomainManager) CancelVMIMigration(vmi *v1.VirtualMachineInstance) error {
+	domName := api.VMINamespaceKeyFunc(vmi)
+	dom, err := l.virConn.LookupDomainByName(domName)
+	if err != nil {
+		log.Log.Object(vmi).Reason(err).Warning("failed to cancel migration, domain not found ")
+		return err
+	}
+	stats, err := dom.GetJobInfo()
```

Comment:
i think i asked this in another PR recently, but I can't remember for sure.

if no job is active, will that return an error here, or will we still get a stats obj?

Reply:
It'll return an empty jobInfo object

Reply:
thanks, i'll probably forget and ask yet again next review, lol.